frontend/talentsearch/src/js/lang/fr.json

```diff
@@ -1636,5 +1636,283 @@
+    "description": "Description of pool recruitment, paragraph one"
+  },
+  "l7Hif/": {
+    "defaultMessage": "Compétences sélectionnées",
+    "description": "Section header for a list of skills selected"
+  },
+  "l9AK3C": {
+    "defaultMessage": "ou",
+    "description": "that appears between different experience requirements for a pool applicant"
+  },
+  "ls7b2p": {
+    "defaultMessage": "Plage de salaire :",
+    "description": "Label for pool advertisement salary range"
+  },
+  "m5hjaz": {
+    "defaultMessage": "Lorsque les responsables du recrutement ont des besoins en personnel informatique et que des postes se libèrent, les candidats qui répondent aux qualifications de ce processus peuvent être contactés pour une évaluation plus approfondie. Cela signifie que divers responsables peuvent vous contacter au sujet d'occations précises dans le domaine du développement d'applications.",
+    "description": "Description of pool recruitment, paragraph two"
+  },
+  "mNWpoy": {
+    "defaultMessage": "Détails supplémentaires",
+    "description": "Title for the details section of a pool advertisement"
+  },
+  "mbtf3h": {
+    "defaultMessage": "Pour être admis dans ce processus, vous devrez soumettre des suffisamment d’informations pour vérifier votre expérience dans <strong>toutes ces compétences (Exigences - Professionnelles)</strong> avec votre candidature",
+    "description": "Explanation of a pools required occupational skills"
+  },
+  "ojN4Si": {
+    "defaultMessage": "Date de clôture :",
+    "description": "Label for pool advertisement closing date"
+  },
+  "owzveI": {
+    "defaultMessage": "Expérience et formation",
+    "description": "Title for experience and education pool requirements"
+  },
+  "qNL/Rp": {
+    "defaultMessage": "Apprentissage pendant l'emploi",
+    "description": "pool experience requirement, on job learning"
+  },
+  "r9FSaq": {
+    "defaultMessage": "Réussite de deux ans d'études postsecondaires en informatique, en technologie de l'information, en gestion de l'information ou dans une autre spécialité pertinente pour ce poste",
+    "description": "post secondary education experience for pool advertisement"
+  },
+  "rYodJu": {
+    "defaultMessage": "Durée indéterminée (permanente)",
+    "description": "Duration that is permanent"
+  },
+  "s60QyR": {
+    "defaultMessage": "2 ans ou plus d'expérience combinée dans un domaine connexe, y compris l'un des éléments suivants :",
+    "description": "lead in to list of experience required for a pool applicant"
+  },
+  "uv2lY0": {
+    "defaultMessage": "Votre travail",
+    "description": "Title for a pool advertisements key tasks section."
+  },
+  "vmyStM": {
+    "defaultMessage": "Par mot-clé",
+    "description": "Button text for the search skills tab on skills filter"
+  },
+  "wuC7Tj": {
+    "defaultMessage": "Si ce processus semble vous convenir, postulez dès maintenant!"
```

Comment:
Is it possible to add a description to the English string so that one can find the English string if necessary?